boards/arc/nsim/board.cmake

```diff
@@ -9,6 +9,9 @@ endif()
 if(NOT ${CONFIG_SOC_NSIM_HS_SMP})
 board_set_flasher_ifnset(arc-nsim)
```

Comment:
While we adding MDB runner support for all ARC platforms why skipping UP nSIM? This might in fact be very useful at least for making sure MDB runner really works as it's much simpler than SMP setup.

Reply:
ok, i will add it.